ChangeLog

```diff
@@ -1,3 +1,75 @@
+* Thu Oct 15 2020 Chris Lumens <clumens@redhat.com> Pacemaker-2.0.5
+- Changesets: 497
+- Diff: 285 files changed, 22704 insertions(+), 14458 deletions(-)
+
+- Features added since Pacemaker-2.0.4
+  + cibsecret: read value to set from input (or stdin) if not specified
+  + controller: add new IPC API command for getting node list
+  + libpe_status: Add type="integer" to rule elements, allowing for
+				  specifying 64-bit integers.
+  + scheduler: Add rule-based tests to op_defaults and rsc_defaults.
+  + scheduler: Add on-fail=demote and no-quorum-policy=demote recovery
+               policies for promoted resources.
+  + tools: Add --resource= to filter crm_mon output for a resource.
+  + tools: Add -P to crmadmin to ping via pacemakerd-api.
+  + tools: Use glib for cmdline handling in crmadmin and crm_resource.
+
+- Fixes for regressions introduced in Pacemaker-2.0.4
+  + scheduler: Add the node name back to bundle instances.
+  + tools: get stonith_admin --list-targets working again
+
+- Fixes for regressions introduced in Pacemaker-2.0.3
+  + scheduler: Fix adding HTTP headers to crm_mon in daemon mode.
+
+- Fixes for regressions introduced in Pacemaker-2.0.1
+  + scheduler: require pre-/post-start notifications correctly.
+
+- Changes since Pacemaker-2.0.4
+  + CIB: use crm_exit to free qb-logging
+  + cibsecret: don't use pssh -q option unless supported
+  + controller: Log correct timeout for timed-out stonith monitor
+  + controller: properly detect remote node info requests
+  + daemons: Accept PCMK_OCF_DEGRADED and PCMK_OCF_DEGRADED_MASTER status codes
+  + daemons: Notify tengine of completion of CRM_OP_LRM_DELETE.(CLBZ:#5424)
```

Comment:
Something like "controller: avoid recovery delay when shutdown locks expire"

FYI it's very difficult to write changelog entries without having been involved in most of the PRs. Developers naturally write commit messages about implementation, whereas the changelog needs to have the user-visible effects. Sometimes the full commit message gives enough context, otherwise sometimes the PR description does, and it's usually not worth the time chasing it down beyond that.